docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -268,4 +268,32 @@ The new indexes are both applied to the `RESOURCE_SERVER_PERM_TICKET` table. If
 {project_name} will skip the creation of the indexes by default during the automatic schema migration, and will instead log the SQL statements
 on the console during migration. In this case, the statements must be run manually in the DB after {project_name}'s startup.
 
-See the link:{upgradingguide_link}[{upgradingguide_name}] for details on how to configure a different limit.
\ No newline at end of file
+See the link:{upgradingguide_link}[{upgradingguide_name}] for details on how to configure a different limit.
+
+= Removing deprecated methods from `AccessToken`, `IDToken`, and `JsonWebToken` classes
+
+The following methods were removed from the `AccessToken` class:
+
+* `expiration`. Use the `exp` method instead.
+* `notBefore`. Use the `nbf` method instead.
+* `issuedAt`. Use the `iat` method instead.
+
+The following methods were removed from the `IDToken` class:
+
+* `getAuthTime` and `setAuthTime`. Use the `getAuth_time` and `setAuth_time` methods, respectively.
+* `notBefore`. Use the `nbf` method instead.
+* `issuedAt`. Use the `iat` method instead.
+
```

Comment:
Is it possible to add also this line on l286 :
```
* `setSessionState`. Use the `setSessionId` method instead (See the details above in the section about `session_state` claim)
```

The method `IDToken.setSessionState()` was not removed in your PR, but will makes sense to mention it here in this section as well to have all the "removals" from `IDToken` in Keycloak 25 mentioned in this section. (But existing section about `setSessionState` can be kept as is IMO as it contains some additional changes related to `session_state` claim).

Reply:
Done